index.js

```diff
@@ -15,6 +19,20 @@ module.exports = {
     }
     // eslint-disable-next-line no-unused-expressions
     this._super.init && this._super.init.apply(this, arguments)
-  }
+  },
   /* eslint-enable complexity */
+
+  included () {
+    this._super.included.apply(this, arguments)
+    this.import('vendor/echarts-en.min.js')
+    this.import('vendor/shims/echarts.js')
+  },
+
+  treeForVendor (vendorTree) {
+    const momentTree = new Funnel(path.join(this.project.root, 'node_modules', 'echarts', 'dist'), {
```

Comment:
momentTree?

Reply:
I'll rename to echartTree. Was following a moment guide of exposing npm libraries as a shim